python-threatexchange/threatexchange/extensions/video_vpdq/video_vpdq.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Wrapper around the vpdq signal type.
+"""
+
+import vpdq  # type: ignore
+from .vpdq_util import json_to_vpdq, vpdq_to_json
+from .vpdq_brute_hasher import match_VPDQ_hash_brute
+import pathlib
+import typing as t
+from threatexchange.content_type.content_base import ContentType
+from threatexchange.content_type.video import VideoContent
+import re
+from threatexchange.signal_type import signal_base
+
+
+class VideoVPDQSignal(signal_base.SimpleSignalType, signal_base.TextHasher):
+    """
+    Simple signal type for video using VPDQ.
+    Read about VPDQ at https://github.com/facebook/ThreatExchange/tree/main/vpdq
```

Comment:
blocking: Describe also the serialization for the hashes, or a reference to how they are stored